Xamarin.LibZipSharp.targets

```diff
@@ -0,0 +1,26 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <LibZipSharpBundleAllNativeLibraries Condition=" '$(LibZipSharpBundleAllNativeLibraries)' == '' ">false</LibZipSharpBundleAllNativeLibraries>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)..\lib\net45\libZipSharp.dll.config">
+      <Link>libZipSharp.dll.config</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <_LibZipNativeLibs Include="$(MSBuildThisFileDirectory)..\runtimes\win-x64\native\libzip.dll">
+      <Link>lib64\%(FileName)%(Extension)</Link>
+    </_LibZipNativeLibs>
+    <_LibZipNativeLibs Include="$(MSBuildThisFileDirectory)..\runtimes\win-x86\native\libzip.dll">
+      <Link>%(FileName)%(Extension)</Link>
+    </_LibZipNativeLibs>
+    <_LibZipNativeLibs Include="$(MSBuildThisFileDirectory)..\runtimes\osx\native\libzip.5.0.dylib">
+      <Link>%(FileName)%(Extension)</Link>
+    </_LibZipNativeLibs>
+    <None Include="@(_LibZipNativeLibs)" Condition="'$(LibZipSharpBundleAllNativeLibraries)' == 'True'">
```

Comment:
As discussed on Slack, we'll need a Linux `libzip.so` here as well for netstandard to work.